packages/core/src/helpers.ts

```diff
@@ -1,8 +1,13 @@
 import axios from 'axios';
 import assert from 'assert';
 
-let sentryCaptureException: any = (error) => {
-  console.error(error); 
+let sentryCaptureException: any = error => {
+  console.error(error);
```

Comment:
i know its not changed in this PR, but should this be submitting to sentry?

Reply:
the wallet app will inejct the react-native sentry implementation

Reply:
let me add a comment on this line

Reply:
it should be updated now